src/osemgrep/core/Engine_type.ml

```diff
@@ -3,7 +3,11 @@
    about configuring the engine, whereas Engine_kind.t is more useful as metrics for the users.
 *)
 
-type analysis_flavor = Intraprocedural | Interprocedural | Interfile
+type analysis_flavor =
+  | Intraprocedural
+  | Interprocedural
+  | Interfile
+  | GitRemote of string
```

Comment:
nah, this should not be part of type. This type is to declare which kind of program analysis we do; The location of the target file has nothing to do with it.

Reply:
Yea fair